packages/core/package.json

```diff
@@ -47,26 +34,22 @@
     "url": "https://github.com/davidkpiano/xstate/issues"
   },
   "homepage": "https://github.com/davidkpiano/xstate/tree/master/packages/core#readme",
+  "dependencies": {
```

Comment:
This had to be included because `xstate` imports it from within `src/scxml.ts`. I bet you don't want to have this here - but we'd have to restructure things in some followup PR to get rid of this.

This actually has showcased a really nice validation provided by preconstruct - it has validated, during build, if your package.json declares all imported dependencies.

Reply:
Could this be a dev-dependency?

Reply:
Nope, it's currently part of the core package and has to declared as either dependency or peer dependency.

It's because `@xstate/scxml` imports `toMachine` here:
https://github.com/davidkpiano/xstate/blob/53bed804ec90e9c2821ed5dcd9d1b9443b8ae533/packages/xstate-scxml/test/scxml.test.ts#L7 which is depending on this `xml-js` package.

This is not a big deal though - we only need to restructure things a little bit. Most likely `core/src/scxml.ts` should just be moved to `@xstate/scxml` package and then we could remove this dependency from here (because it would be moved to the other package). This should in a followup PR - if you have any other idea on how to restructure those things then let's discuss this on Slack.